jsapp/js/actions.es6

```diff
@@ -512,6 +515,27 @@ actions.resources.removeSubmissionValidationStatus.listen((uid, sid) => {
   });
 });
 
+actions.resources.deleteSubmission.listen((uid, sid) => {
+  dataInterface.deleteSubmission(uid, sid).done(() => {
+    notify(t('submission deleted'));
+    actions.resources.deleteSubmission.completed();
+    actions.resources.loadAsset({id: uid});
+  });
```

Comment:
Oh, please add a `.fail(actions.resources.deleteSubmission.failed)` here :) I know you don't use it, but it would be expected to be here